Description: Default closer to install-guide
 In order to simplify the install procedure in the install-guide, this patch
 makes defaults closer to what the guide is proposing, making the customization
 a lot more easy and fast.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-10-23

--- neutron-2014.2.orig/etc/l3_agent.ini
+++ neutron-2014.2/etc/l3_agent.ini
@@ -19,7 +19,7 @@ interface_driver = neutron.agent.linux.i
 
 # Allow overlapping IP (Must have kernel build with CONFIG_NET_NS=y and
 # iproute2 package that supports namespaces).
-# use_namespaces = True
+use_namespaces = True
 
 # If use_namespaces is set as False then the agent can only configure one router.
 
@@ -42,7 +42,7 @@ interface_driver = neutron.agent.linux.i
 # Name of bridge used for external network traffic. This should be set to
 # empty value for the linux bridge. when this parameter is set, each L3 agent
 # can be associated with no more than one external network.
-# external_network_bridge = br-ex
+external_network_bridge = br-ex
 
 # TCP Port used by Neutron metadata server
 # metadata_port = 9697
--- neutron-2014.2.orig/etc/dhcp_agent.ini
+++ neutron-2014.2/etc/dhcp_agent.ini
@@ -9,11 +9,10 @@
 
 # The DHCP agent requires an interface driver be set. Choose the one that best
 # matches your plugin.
-# interface_driver =
-
+# 
 # Example of interface_driver option for OVS based plugins(OVS, Ryu, NEC, NVP,
 # BigSwitch/Floodlight)
-# interface_driver = neutron.agent.linux.interface.OVSInterfaceDriver
+interface_driver = neutron.agent.linux.interface.OVSInterfaceDriver
 
 # Name of Open vSwitch bridge to use
 # ovs_integration_bridge = br-int
@@ -28,11 +27,11 @@
 
 # The agent can use other DHCP drivers.  Dnsmasq is the simplest and requires
 # no additional setup of the DHCP server.
-# dhcp_driver = neutron.agent.linux.dhcp.Dnsmasq
+dhcp_driver = neutron.agent.linux.dhcp.Dnsmasq
 
 # Allow overlapping IP (Must have kernel build with CONFIG_NET_NS=y and
 # iproute2 package that supports namespaces).
-# use_namespaces = True
+use_namespaces = True
 
 # The DHCP server can assist with providing metadata support on isolated
 # networks. Setting this value to True will cause the DHCP server to append
